x/cosmwasm/README.md

```diff
@@ -0,0 +1,34 @@
+# CosmWasm support
+
+This module contains CosmWasm integration points.
+
+This module provides first class support for:
```

Comment:
s/module/package/ various places. 

Module is an interface with a particular meaning in the Cosmos SDK.